### PR TITLE
Add second page about education

### PR DESCRIPTION
**Pull request by dash102**
These changes build on top of #1 with a page about coursework.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,20 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
Comment (hlz03) [inline portfolio/src/main/webapp/script.js:14]: add a function doc to describe this function.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,20 +12,19 @@
...
- * Adds a random fact to the page.
- */
-function addRandomFact() {
-  const facts =
-      ["I used to be a springboard diver until I hit my face on the diving board :(", 
-      "Zhajiangmian is my favorite food!", 
-      "I'm from Kansas!", 
-      "I'm a huge Marvel fan!"];
+function openCourses(evt, courseName) {
+    var i, tabcontent, tablinks;
```
Comment (hlz03) [inline portfolio/src/main/webapp/script.js:16]: instead, use for (let i = 0, ...) directly.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -65,3 +91,38 @@ body, html {
 }
 
 /* End social bar styling */
+
+/* Begin coursework styling */
+
```
Comment (hlz03) [inline portfolio/src/main/webapp/style.css:96]: remove the empty line.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (dash102) [reply portfolio/src/main/webapp/script.js:14]: Done

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (dash102) [reply portfolio/src/main/webapp/script.js]: Done

**Comment by dash102** [comment]
This PR should only have commits relating to the education page now (reformatting commit was taken out)

**Pull request merged**